community/cypher/cypher-compiler-3.3/src/test/scala/org/neo4j/cypher/internal/compiler/v3_3/planner/logical/plans/rewriter/SimplifyPredicatesTest.scala

```diff
@@ -42,6 +43,18 @@ class SimplifyEqualityTest extends CypherFunSuite with LogicalPlanningTestSuppor
     val orgPredicate: Expression = In(Property(varFor("x"), PropertyKeyName("prop")(pos))(pos), collection)(pos)
     val selection = Selection(Seq(orgPredicate), singleRow)(solved)
 
-    selection.endoRewrite(simplifyEquality) should equal(selection)
+    selection.endoRewrite(simplifyPredicates) should equal(selection)
+  }
+
+  test("should rewrite WHERE AndedPropertyInequality(x.prop, 1) to WHERE x.prop <> 1") {
+    val singleRow: LogicalPlan = Argument(Set(IdName("a")))(solved)(Map.empty)
+    val variable = Variable("a")(pos)
```

Comment:
I'm still so bad at reading these manually built logical plans... I guess `"a"` here really is what is called `x` in the test description?